setup.py

```diff
@@ -29,6 +29,7 @@
         "python-stdnum",
         "requests",
         "ruamel.yaml",
+        "scikit-learn",
```

Comment:
since it was never part of the deps so far, and support for this is also fading, `scikit-learn` could for now be added to the test dependencies only.